lighthouse-core/lib/i18n/i18n-module.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
```

Comment:
```
File '/Users/cjamcl/src/lighthouse/lighthouse-core/lib/i18n/swap-locale.js' is not under 'rootDir' '/Users/cjamcl/src/lighthouse/report'. 'rootDir' is expected to contain all source files
```

@brendankenny what do?

Reply:
![image](https://user-images.githubusercontent.com/4071474/134254605-d9bd6583-6935-4567-bde4-e3f672db934a.png)
nah that just moves the error.

How do I include code across ts projects?

Reply:
> How do I include code across ts projects? should the `lib/i18n` be its own project?

it's by far the easiest way, and doing that also makes bundling more straightforward because we deal with/sever most of the interdependencies in the source files instead of the bundling code

#13120 (which you've already seen :)